src/frontend/src/components/Posts/Timeline.jsx

```diff
@@ -44,7 +45,7 @@ const Timeline = ({ pages, nextPage }) => {
   // key each time, based on page (i.e., size), so we remove the previous one
   if (nextPage) {
     postsTimeline.push(
-      <LoadMoreButton onClick={() => nextPage()} key={`load-more-button-${pages.length}`} />
```

Comment:
@klee214 
1)If we are not using `LoadMoreButton` anymore delete it from our components folder.

Reply:
Sure! I just left the component there just in case! I will delete the file!